packages/app-store/_utils/getEventTypeAppData.ts

```diff
@@ -18,12 +18,14 @@ export const getEventTypeAppData = <T extends EventTypeAppsList>(
     return allowDataGet
       ? {
           ...appMetadata,
+          // We should favor eventType's price and currency over appMetadata's price and currency
+          price: eventType.price || appMetadata.price,
+          currency: eventType.currency || appMetadata.currency,
```

Comment:
Since adding an app to an event type was saving default wrong data we should favour eventType price and currency and not from metadata.